packages/zk-transfer-web/src/App.tsx

```diff
@@ -16,24 +18,42 @@ type AppProp = {
 function App({ demo } : AppProp) {
   const [state, setState] = useState({
     chainId: demo.chainId,
-    screenName: ''
+    screenName: "",
+    ZKToken: new Map(),
   });
 
   useEffect(() => {
     const { chainId } = state;
 
     if (!chainId) {
       demo.clientInit()
-        .then(() => {
+        .then(async () => {
+          console.groupCollapsed('Client Initialized');
+          const api = await makeApi(demo);
+          //@ts-ignore
+          window.api = api; window.demo = demo;
```

Comment:
I'm not sure this is the right place to put it.
Doesn't webpack already populate `window.api` and `window.demo` ?